dts/arm/renesas/gen3/r8a77951.dtsi

```diff
@@ -5,3 +5,30 @@
  */
 
 #include <renesas/gen3/rcar_gen3_cr7.dtsi>
+#include <dt-bindings/pinctrl/renesas/pinctrl-r8a77951.h>
+
+&pfc {
+	can0_data_a_tx: can0_data_a_tx {
```

Comment:
note: pre-defined nodes are, in general, usable for a single state (e.g. `default`). If you plan to add support for other states in the future (e.g. `sleep`), you may want to encode the state name on the node label. Note that it may not always be the case: sleep state for a pin could be represented independently of the mapped signal, e.g. `uart0_tx_pa0`, `spi1_miso_pa0`  for `default` and `pa0_sleep` for `sleep` in any case. See https://docs.zephyrproject.org/latest/guides/pinctrl/index.html#pin-configuration for more details.